docs/modules/ROOT/pages/user-code-deployment.adoc

```diff
@@ -1,6 +1,8 @@
 = User Code Deployment
 :description: Hazelcast allows you to deploy your own custom and domain classes to cluster members. From Hazelcast Platform Operator, you can deploy your application code from external buckets and ConfigMap objects in Kubernetes.
 
+WARNING: User Code Delpoymen is deprecated. Use xref:user-code-namespaces.adoc[User Code Namespaces] instead.
```

Comment:
```suggestion
WARNING: User Code Delpoyment is deprecated. Use xref:user-code-namespaces.adoc[User Code Namespaces] instead.
```

Reply:
I think it should be *Deployment* 🙂

Reply:
Good catch Hasan! Hey, it is Friday afternoon - suggestion updated

Reply:
Thanks for the catch! Sorry @rebekah-lawrence, I missed that you've posted a suggestion and pushed the change manually 🙂